hunts/src/HuntViewMain.js

```diff
@@ -69,82 +69,84 @@ export const HuntViewMain = (props) => {
   }, [alert.id]);
 
   return (
-    <div
-      style={{
-        margin: "0 20px",
-      }}
-    >
-      <Alert
-        dismissible
-        variant={alert.variant}
-        show={alert.show}
-        onClose={() => dispatch(hideAlert())}
-      >
-        {alert.text}
-      </Alert>
-      <HuntViewHeader hunt={hunt} />
+    <>
```

Comment:
This file is mostly an indentation change as a result of the fragment I added here